tests/test_maudau.py

```diff
@@ -27,7 +27,7 @@ def make_frame(spark):
         ('a', '2017-05-05', '20170507'),
     ]
     return spark.createDataFrame(
-        [dict(zip(cols, tup)) for tup in values],
+        [dict(list(zip(cols, tup))) for tup in values],
```

Comment:
this `list` is unnecessary, the `dict` works just fine in python 2 and 3

Reply:
This is an artifact of the 2to3 conversion, it seems to be liberal in sprinkling lists around all of the iterators.